Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -514,11 +518,14 @@ function Main {
 						$pattern = "INFINIBAND_VERIFICATION_SUCCESS_OMB_P2P_ALLNODES"
 						Write-LogInfo "Analyzing $logFileName"
 						$metaData = "InfiniBand-Verification-$Iteration-$TempName : OMB-P2P"
-						$SucessLogs = Select-String -Path $logFileName -Pattern $pattern
-						if($SucessLogs.Count -eq 1) {
-								$currentResult = $resultPass
+						$SuccessLogs = Select-String -Path $logFileName -Pattern $pattern
```

Comment:
I see repetitive code line #478-485 & # 500-508 & # 521-527
We can have a function to avoid that.

Reply:
Sure, but this PR does not have this change. There is a regression in Ubuntu, and I work on it now. Will make this change in the next PR. Ok?

Reply:
ok. Please make sure it will be reflected in the future.